test/test_centos_fedora_support.py

```diff
@@ -0,0 +1,217 @@
+import pytest
+from conftest import (
+    tick_box,
+    info_box,
+    cross_box,
+    mock_command,
+    mock_command_2,
+)
+
+
+@pytest.mark.parametrize("tag", [('fedora'), ])
+def test_epel_and_remi_not_installed_fedora(Pihole):
+    '''
+    confirms installer does not attempt to install EPEL/REMI repositories
+    on Fedora
+    '''
+    distro_check = Pihole.run('''
+    source /opt/pihole/basic-install.sh
+    distro_check
+    ''')
+    expected_stdout = info_box + (' Enabling EPEL package repository '
+                                  '(https://fedoraproject.org/wiki/EPEL)')
+    assert expected_stdout not in distro_check.stdout
```

Comment:
positive matching is vastly superior to negative matching because `"asdf" not in any.stdout` when doing negative string matches keep it as loose as possible.